docs/reference/announcements/880.md

```diff
@@ -123,25 +123,20 @@ Additional upgrade considerations are necessary for deployments that use custom
 
 ## Camunda Exporter <span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span>
 
-A new Camunda Exporter brings the importer and archiving logic of web components (Tasklist and Operate) closer to the distributed platform (Zeebe). The index schema is also being harmonized.
+Our web applications have historically used importers and archivers to consume, aggregate, and archive historical data provided by our Elasticsearch (ES) or OpenSearch (OS) Exporter.
 
-The exporter can consume Zeebe records (mostly events created by the engine), aggregate data, and store the related data into shared and harmonized indices.
+![current-state](../img/current-state-refined.png)
```

Comment:
I'd avoid using `current` as it doesn't refer to which version it is.

Reply:
I dont see it as an issue. It is in an announcement state for the 8.8 release. But for sure it can be renamed as well.

Reply:
Done.